web_src/js/features/citation.ts

```diff
@@ -30,6 +30,7 @@ export async function initCitationFileCopyContent() {
   const citationCopyApa = document.querySelector<HTMLButtonElement>('#citation-copy-apa');
   const citationCopyBibtex = document.querySelector<HTMLButtonElement>('#citation-copy-bibtex');
   const inputContent = document.querySelector<HTMLInputElement>('#citation-copy-content');
+  const modal = fomanticQuery('#cite-repo-modal');
```

Comment:
`$modal`

Reply:
Done in 4373ad1a623e9756507cd6356edf388017f3b239